index.js

```diff
@@ -0,0 +1,17 @@
+var path = require('path');
+
+var tabzillaEntryPoint = require.resolve('mozilla-tabzilla');
+var tabzillaDir = path.join(path.dirname(tabzillaEntryPoint), 'css');
+
+function includePaths() {
+  return tabzillaDir;
+}
+
+module.exports = {
+  includePaths: includePaths(),
+  with: function() {
+    var paths  = Array.prototype.slice.call(arguments);
+    var result = [].concat.apply(includePaths(), paths);
```

Comment:
@schalkneethling No clue. I just copy-pasted the boilerplate from node-normalize-scss and bourbon/neat modules since it seemed to be the same in all 3 projects. So I went with consistency and "prior art" (and laziness).